_data/people/jlefevre.yml

```diff
@@ -36,4 +36,9 @@ presentations:
     meeting: "CHEP 2019"
     meetingurl: https://indico.cern.ch/event/773049/contributions/3474413/
     focus-area: doma
-    
+  - title: "Skyhook Data Management: programmable object storage for databases"
+    date: 2020-05-26
+    url: https://drive.google.com/file/d/11uG0dJxxGfyc4RZ60dChU22nzO-dWL78/view?usp=sharing
+    meeting: "Fujitsu Labs"
+    meetingurl: https://www.fujitsu.com/jp/group/labs/en/
+    focus-area: doma
```

Comment:
Is there a project label to add to this entry? (eg, for Skyhook)

Reply:
Added project label to each entry.